### ISSUE TITLE
Trakt API Limit

### ISSUE THREAD

**Issue by devregnfo** [jewbmx/repo#50]
Hi,
I was wondering if we can add our own TRAKT API key somewhere, so the errors of limit exceeded are avoided...?
Is it hardcoded or can we change some config?
The settings only allow for activation/autorization but no placeholder for the key

Thanks

**Comment by devregnfo** [comment]
Weel, after digging a bit I found the coulprit. 
trakt.py
I added my own API key and reactivated with trakt. All good.

But when I tried to continue watching the show I was on... NO SOURCES.
And other shows, have little to no sources and nothing is playing.
Even after reseting the box and the router.

Can someone explain what the hardcoded API trakt key has to do with the sources I'm getting? I use VPN service as well...
Or is it just a glitch and some random f*** coincidence?

Thanks
